compose.go

```diff
@@ -76,6 +76,56 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
```

Comment:
Would be great for compose:add to check if url of package is already present or not in plasma-compose.yaml. At the moment, same package can be added multiple times:
```
name: plasma
dependencies:
    - name: plasma-core
      source:
        type: git
        url: https://projects.skilld.cloud/skilld/pla-plasma.git
        tag: 1.23.0
    - name: plasma-core
      source:
        type: git
        url: https://projects.skilld.cloud/skilld/pla-plasma.git
        ref: master
        tag: 1.23.0
```